src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,39 @@
+@use '../utils/variables' as *;
+
+.stars {
+  display: flex;
+  align-items: center;
+  width: 166px;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-size: 13px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &--0 &__star:nth-child(-n + 6),
+  &--1 &__star:nth-last-child(-n + 5),
+  &--2 &__star:nth-last-child(-n + 4),
+  &--3 &__star:nth-last-child(-n + 3),
+  &--4 &__star:nth-last-child(-n + 2) {
+    background-image: url(/src/images/star.svg);
+  }
+
+  &--1 &__star:nth-child(-n + 1),
+  &--2 &__star:nth-child(-n + 2),
+  &--3 &__star:nth-child(-n + 3),
+  &--4 &__star:nth-child(-n + 4),
+  &--5 &__star:nth-child(-n + 5) {
+    background-image: url(/src/images/star-active.svg);
+  }
```

Comment:
dont know for what you need this, all starts should be active and only last not, so change it

Reply:
Hi. One of the requirements of this task is that all of stars--N modifiers should work as expected so that's why all modifiers are there. Should I delete them?